tests/std_functions_tests/test_wood_brie.py

```diff
@@ -75,3 +80,41 @@ def test_wood_brie_mix(self):
         )
         np.testing.assert_almost_equal(k_w, k_w_ref)
         np.testing.assert_almost_equal(k_b / 1e9, k_b_ref)
+
+
+s_gi = 0.0
+s_oi = 0.5
+s_wi = 0.5
+oil_init_k = 492032461.8739312
+gas_init_k = 208287817.8755032
+brine_init_k = 2814732817.889945
+expected_value = 837640292.3902907
```

Comment:
```suggestion
oil_init_k = 492_032_461.8739312
gas_init_k = 208_287_817.8755032
brine_init_k = 2_814_732_817.889945
expected_value = 837_640_292.3902907
```
Long numbers are more readable when underscores are added. The initial oil value is rather low, perhaps it should be calculated with more realistic numbers.

Reply:
Agree with the underscoring, makes it more readable. I think i copied these values from a test in the `rock-physics` that was using the `multi_wood` function. So if you have more sensible parameter values i'd be happy to replace them :)